install-dev/classes/controllerHttp.php

```diff
@@ -242,6 +242,12 @@ public function setCurrentStep($step)
             if (self::$steps->current()->getControllerInstance()->validate()) {
                 self::$steps->next();
             }
+
+            // Don't display system step if mandatory requirements is valid
+            if (self::$steps->current()->getName() == 'system' && self::$steps->current()->getControllerInstance()->validate()) {
```

Comment:
There is actually something I do not understand. How adding the same check for a specific step will solve the described issue?
There's probably an issue in the first `validate()` call which prevent the step to be properly validated. What do you think?

Reply:
Had to find it by myself, but the code could be previously found in https://github.com/PrestaShop/PrestaShop/commit/71717b89b619b13ee85045490433df873b6924fb#diff-314f7a40b328a8f143131097e1a37743L325